src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -110,7 +113,7 @@ def observe_release(project_id: int, release_id: int, environment: Optional[str]
     return release_observed == "1"  # type: ignore
 
 
-def get_boosted_releases(project_id: int) -> List[Tuple[int, Optional[str], float]]:
+def get_and_expire_boosted_releases(project_id: int) -> List[Tuple[int, Optional[str], float]]:
```

Comment:
new name confuse me much more, i see that this function does 2 things at time.
func doc has explanation and i feel let's keep old name.
ideally we can refactor this code during another ticket for latest releases.